vale/styles/jargonLint/Licenses.yml

```diff
@@ -45,12 +48,36 @@ swap:
   CC[ -]?BY[ -]?SA[ -]?4(?:\.0)?: CC BY-SA 4.0
   CC[ -]?[O0]: CC0
   CC[ -]?0[ -]?1(?:\.0)?: CC0 1.0
-  # GNU Licences
+  # GNU licences
   AGPL: AGPL
   AGPL[ -]?v?3(?:\.0)?: AGPLv3
   GPL(?!\.org): GPL
   GPL[ -]?v?2(?:\.0)?: GPLv2
-  GPL[ -]?v?3(?!\+): GPLv3
-  GPL[ -]?v?3\+: GPLv3+
+  GPL[ -]?v?3(?!\.0)?: GPLv3
   LGPL: LGPL
+  LGPL[ -]?v?2(?:\.1)?: LGPLv2.1
   LGPL[ -]?v?3(?:\.0)?: LGPLv3
+exceptions:
+  # Exceptions to avoid correcting SPDX short identifiers
```

Comment:
I would have expected to see them all added to the .good.txt

Reply:
Please check the new test files.